src/main/kotlin/org/dashevo/dpp/contract/DataContract.kt

```diff
@@ -12,18 +12,23 @@ import org.dashevo.dpp.errors.InvalidDocumentTypeError
 
 class DataContract(var id: String,
                    var ownerId: String,
+                   val protocolVersion: Int,
                    var schema: String,
                    var documents: MutableMap<String, Any?>,
                    var definitions: MutableMap<String, Any?> = hashMapOf()) : BaseObject() {
 
     companion object DEFAULTS {
         const val SCHEMA = "https://schema.dash.org/dpp-0-4-0/meta/data-contract"
+        const val PROTOCOL_VERSION: Int = 0
     }
 
     var entropy: String? = null
 
     constructor(rawContract: MutableMap<String, Any?>) : this(rawContract["\$id"] as String,
             rawContract["ownerId"] as String,
+            if (rawContract.containsKey("protocolVersion"))
+                rawContract["protocolVersion"] as Int
+            else 0,
```

Comment:
In this case `0` means the same as `PROTOCOL_VERSION` or we want to set the value strictly to `0`. If it means the latest `PROTOCOL_VERSION` as default, we should probably use the cosnt from above.

Reply:
Good point.  This will be a bug later and is not the right way to handle this.

Reply:
https://github.com/dashevo/android-dpp/pull/9/commits/e5fb399bc3a9284e2417521069104518d63faa5b - fixed here.

Reply:
https://github.com/dashevo/android-dpp/pull/9/commits/e5fb399bc3a9284e2417521069104518d63faa5b - fixed here